resources/tools/index.md

```diff
@@ -0,0 +1,90 @@
+---
+layout: custom
+title: titles.tools
+permalink: /resources/tools/index.html
+---
+
+<div class="text-center container description">
+    <p>
+        {% t tools.intro1 %}
+        <a href="https://github.com/monero-project/monero-site/issues" target="_blank" rel="noreferrer noopener">{% t tools.intro2 %}</a>.
+    </p>
+</div>
+
+<section class="container tools">
+    <div class="row">
+        <div class="left half no-pad-sm col-lg-6 col-md-6 col-sm-12 col-xs-12">
+            <div class="info-block">
+                <div class="row">
+                    <div class="col">
+                        <h2>{% t tools.block-explorers %}</h2>
+                        <p><a href="https://xmrchain.net/">xmrchain.net</a></p>
+                        <p><a href="https://moneroblocks.info">MoneroBlocks</a></p>
+                        <p><a href="https://www.exploremonero.com/">Explore Monero</a></p>
+                        <p><a href="https://monerovision.com">MoneroVision</a></p>
+                    </div>
+                </div>
+            </div>
+        </div>
+        <div class="right half col-lg-6 col-md-6 col-sm-12 col-xs-12">
+            <div class="info-block">
+                <div class="row">
+                    <div class="col">
+                        <h2>{% t tools.address-generators %}</h2>
+                        <p><a href="https://moneroaddress.org/">moneroaddress.org</a></p>
+                        <p><a href="https://xmr.llcoins.net/">xmr.llcoins.net</a></p>
+                    </div>
+                </div>
+            </div>
+        </div>
+    </div>
+    <div class="row">
+        <div class="left half col-lg-6 col-md-6 col-sm-12 col-xs-12">
+            <div class="info-block">
+                <div class="row">
+                    <div class="col">
+                        <h2>{% t tools.network %}</h2>
+                        <p><a href="https://monerohash.com/nodes-distribution.html">monerohsah.com - {% t tools.monerohash-nodes %}</a></p>
+                        <p><a href="https://moneroblocks.info/stats">MoneroBlocks - {% t tools.moneroblocks-stats %}</a></p>
+                        <p><a href="https://nownodes.io/connect-xmr-node">NOWNodes - {% t tools.nownodes %}</a></p>
```

Comment:
I think this is just an API provider. I don't see any standalone block explorer on this page.

Reply:
That's why I put it under 'Network.' Are you suggesting moving it to 'Miscellaneous'?

Reply:
Yeah, that's probably better.

Reply:
I just realized:
* did you mean just to update the description in `en.yml` to not include the 'Explorer' part?
* if we are going to move this one to 'Miscellaneous', I'll also move `getblock.io` to there as well since that's also a node provider?

Reply:
Friendly ping @erciccione :)
Just wanted clarification on this move

Reply:
> did you mean just to update the description in en.yml to not include the 'Explorer' part?

Yeah, the 'Explorer' part should be removed.

> if we are going to move this one to 'Miscellaneous', I'll also move getblock.io to there as well since that's also a node provider?

Right. I think makes more sense to leave them where they are then.